v63003/inref/gtm8844.m

```diff
@@ -11,23 +11,23 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;
 haltfn
-	set $etrap="do incrtrap^incrtrap"
+	set $ztrap="goto incrtrap^incrtrap"
 	write "# Halt 1, Expect a RESTRICTEDOP error",!
 	halt
-	hang:$ZTRNLNM("pause") 1
+	hang:$ZTRNLNM("pause") .5
 	write "# Halt 2",!
 	halt
-	write "Haltfn failed"
+	hang 1
```

Comment:
Add a comment that this hang is needed to avoid another RESTRICTEDOP error when the quit happens (which does an implicit ZHALT).